components/courses/app/controllers/admin/courses/drop_controller.rb

```diff
@@ -1,6 +1,8 @@
 module Admin
   module Courses
     class DropController < ::Admin::Courses::BaseController
+      skip_before_action :execute_show_tab_policy
```

Comment:
There is corresponding before_action in Admin::Courses::BaseController, but it's needed only for controllers, which have tabs in menu. That's why I came up with this solution for other controllers. Is there any better way to do this?